WPFHospitalEditor/HospitalMap.xaml.cs

```diff
@@ -21,25 +26,30 @@ public partial class HospitalMap : Window
         IMapObjectController mapObjectController = new MapObjectController();
         IEquipmentTypeServerController equipmentTypeServerController = new EquipmentTypeServerController();
         IMedicationServerController medicationServerController = new MedicationServerController();
+        IDoctorServerController doctorServerController = new DoctorServerController();
+        ISchedulingServerController schedulingController = new SchedulingServerController();
 
         public static Canvas canvasHospitalMap;  
         public static Role role;
         public static List<MapObject> searchResult = new List<MapObject>();
         public static List<EquipmentDto> equipmentSearchResult = new List<EquipmentDto>();
         public static List<MedicationDto> medicationSearchResult = new List<MedicationDto>();
-
+        public static List<RecommendationDto> appointemntSearchResult = new List<RecommendationDto>();
```

Comment:
There is typo in appointemntSearchResult :)

Reply:
Fixed